dbms/src/Server/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@
 # that contain multiple execution modes (client, server, etc.)
 # each of them is built and linked as a separate library, defined below.
 
-add_library(clickhouse-server
+add_library (libclickhouse-server
```

Comment:
Doesn't it will lead to idiosyncratic names like `liblibclickhouse-server.a`?
If so, better to name it as clickhouse-server-lib.